aptos-move/framework/aptos-framework/sources/Token.move

```diff
@@ -63,19 +63,33 @@ module AptosFramework::Token {
         amount: u64,
     }
 
+    /// create collection event with creator address and collection name
+    struct CreateCollectionEvent has drop, store {
+        creator: address,
+        collection_name: ASCII::String,
+    }
+
+    /// token creation event id of token created
+    struct CreateTokenEvent has drop, store {
+        id: TokenId,
+        max: Option<u64>,
+    }
+
     //
     // Core data structures for creating and maintaining tokens
     //
 
-    /// A token creator may have many tokens and many collections of tokens
+    /// Represents collection and token metadata for a creator
     struct Collections has key {
         collections: Table<ASCII::String, Collection>,
         token_data: Table<TokenId, TokenData>,
         burn_capabilities: Table<TokenId, BurnCapability>,
         mint_capabilities: Table<TokenId, MintCapability>,
+        create_collection_events: EventHandle<CreateCollectionEvent>,
+        create_token_events: EventHandle<CreateTokenEvent>,
     }
 
-    /// The source of Tokens, their collection!
+    /// Represent the collection metadata
```

Comment:
nit: s/Represent/Represents to be consistent with the similar comment on line 82.